pkg/cluster/gvk.go

```diff
@@ -0,0 +1,37 @@
+package cluster
```

Comment:
Wait a bit. Missed in the previous change. It's a small misunderstanding in our conversation about namespacing. I meant that it is supposed to be in "gvk" subpackage to be in a separate namespace, not just a file. So all the GVKs namespaced in their own package, not mixing with "cluster". Then additional import of "github.com/opendatahub-io/opendatahub-operator/v2/pkg/cluster/gvk" will be required for the users and access with gvk.Kind (like before for Odh*)

Objections?

Reply:
like this https://github.com/opendatahub-io/opendatahub-operator/pull/981/commits/a8ab4afbe5ff326e0ae25b9d488f18a0973afdbf ?

Reply:
> like this [a8ab4af](https://github.com/opendatahub-io/opendatahub-operator/commit/a8ab4afbe5ff326e0ae25b9d488f18a0973afdbf) ?

Yes, I meant like that